test/integration/api-conversion/00-assert.yaml

```diff
@@ -1,11 +1,11 @@
-apiVersion: lifecycle.keptn.sh/v1alpha1
+apiVersion: lifecycle.keptn.sh/v1alpha3
```

Comment:
why did you change this ?

Reply:
hey @odubajDT, will fix it and other tests too